runtime/compiler/optimizer/SequentialStoreSimplifier.cpp

```diff
@@ -1655,7 +1655,8 @@ bool TR_arraycopySequentialStores::insertConsistentTree()
       dumpOptDetails(comp(), " insertTree: multiplier must be 1 in aiadd tree\n");
       return false;
       }
-   TR::SymbolReference* activeBaseRef = _activeAddrTree->getBaseVarNode()->isNull() ? NULL : _activeAddrTree->getBaseVarNode()->getChild()->skipConversions()->getSymbolReference();
+   TR::Node* activeBaseNode = _activeAddrTree->getBaseVarNode()->isNull() ? NULL : _activeAddrTree->getBaseVarNode()->getChild()->skipConversions();
+   TR::SymbolReference* activeBaseRef = activeBaseNode ? activeBaseNode->getSymbolReference() : NULL;
```

Comment:
In general, it is good practice to call `hasSymbolReference` first before calling `getSymbolReference` 

That pattern is seen in many other places.

Reply:
Fixed. Made an if-block to remove redundant checks.